models/models.js

```diff
@@ -1,5 +1,30 @@
 const db = require('../db/connection');
 
+// --== Models ==--
 exports.fetchTopics = () => {
-  return db.query('select * from topics').then(({ rows }) => rows);
+  console.log('invoked fetchTopics');
+  return db.query('select * from topics;').then(({ rows }) => rows);
+};
+
+exports.fetchArticles = () => {
+  console.log('invoked fetchArticles');
+  return db
+    .query('select * from articles order by created_at desc;')
```

Comment:
Please use sql convention of keywords as uppercase such as SELECT * FROM articles ORDER BY created_at DESC;